docs/data/base/components/tooltip/UnstyledTooltipDelayGroup.tsx

```diff
@@ -8,11 +8,15 @@ export default function UnstyledTooltipDelayGroup() {
       <Tooltip.Provider delay={1000} closeDelay={500}>
```

Comment:
I find this delay too long. I was hovering over the button thinking the demo is broken before noticing it's set for so long. IMO 500ms would be better and readers will still able to notice the lack of delay when moving between buttons.